llvm/lib/Transforms/InstCombine/InstCombineSimplifyDemanded.cpp

```diff
@@ -1959,9 +1959,11 @@ static Constant *getFPClassConstant(Type *Ty, FPClassTest Mask) {
   }
 }
 
-Value *InstCombinerImpl::SimplifyDemandedUseFPClass(
-    Value *V, const FPClassTest DemandedMask, KnownFPClass &Known,
-    unsigned Depth, Instruction *CxtI) {
+Value *InstCombinerImpl::SimplifyDemandedUseFPClass(Value *V,
```

Comment:
Not sure why this changed, the signature didn't?

Reply:
It is needed by `DemandedMask &= ~fcNan;`.